Katello/katello
# Fixes #34472 - add cv empty state action

#### What are the changes introduced in this pull request?
Added empty state action button to the Content Views tab

#### Considerations taken when implementing this change?

#### What are the testing steps for this pull request?
1. Go to the Content -> Content Views tab. 
2. Make sure that you do not have any Content Views to list (have an empty state).
3. "Create Content View" button has to be appear in the middle of the screen. It should open a new modal to let users create a new Content View.
